airflow/operators/python.py

```diff
@@ -486,6 +494,7 @@ def __init__(
         templates_dict: dict | None = None,
         templates_exts: list[str] | None = None,
         expect_airflow: bool = True,
+        skip_exit_code: int | None = None,
```

Comment:
I’m late but this name feels like it’s skipping the exit code; would it be better to name it e.g. `skip_on_exit_code` instead? Also would it be a good idea to allow multiple codes?

Reply:
yes, good suggestions, I'll create a new PR to address them